prepare_spec

```diff
@@ -509,7 +510,20 @@ our $header_name;
     my $thisyear = $ENV{COPYRIGHT_YEAR};
     $thisyear ||= localtime($ENV{SOURCE_DATE_EPOCH} || time)->year() + 1900;
     my @copyrights = @{$self->{copyright} || []};
-    unshift @copyrights, "# Copyright (c) $thisyear SUSE LLC" if $ENV{ADD_SUSE_COPYRIGHT} and not grep /SUSE LLC/, @copyrights;
+    my $add_suse_copyright = $ENV{ADD_SUSE_COPYRIGHT};
+    $add_suse_copyright = 0 if grep /SUSE LLC/, @copyrights;
+    my $changesfile = $specfile;
+    $changesfile =~ s/\.spec$/.changes/;
+    if(!defined($add_suse_copyright) and open(my $fd, "<", $changesfile)) {
+      my $n=0;
+      while(<$fd>) {
+        if(/\w\@suse\.(?:com|cz|de)>/) {
```

Comment:
nope, it only should do that logic for the first email (top entry)

Reply:
If I understand correctly, that addresses the "any" SUSE address question but not the "if anything copyright-worthy has been done" question, right? It seems to me that the copyright should apply for the _last_ email in the changelog and be updated afterward only when a meaningful change has been made.

Reply:
only for the "last email" makes not a lot of sense to me, because thats the initial commit which could be entirely a template and that has no copyrightability in some juristictions (it has in some others).

I'll continue to refrain from commenting publicly on the copyright worthyness.